includes/Data/ContentBlocksResolver.php

```diff
@@ -242,6 +243,33 @@ private static function populate_post_content_inner_blocks( array $block ): arra
 		return $block;
 	}
 
+	/**
+	 * Populates the innerBlocks of this block with navigation item blocks from the referenced navigation post.
+	 *
+	 * @param array<string,mixed> $block The block to populate.
+	 *
+	 * @return array<string,mixed> The populated block.
+	 */
+	private static function populate_navigation_blocks( array $block ): array {
+		if ( 'core/navigation' !== $block['blockName'] || ! isset( $block['attrs']['ref'] ) ) {
```

Comment:
If we are going to rely on a strict types it would be beneficial for us to pay attention when using the undescribed array.
Maybe something like this can prevent from potential notice.

```suggestion
		if ( ($block['blockName'] ?? '') !== 'core/navigation' || ! isset( $block['attrs']['ref'] ) ) {
```

Reply:
@whoami-pwd / @theodesp I strongly disagree with this change, if `blockName` is not set at this point in the lifecycle we have bigger problem that we shouldn't be hiding behind null coercion.

https://github.com/wpengine/wp-graphql-content-blocks/blob/3bbe3e4a033ae664d1a95701bb065a3284acbcf8/includes/Data/ContentBlocksResolver.php#L138C1-L148C33

Reply:
Actually that is a good point 👍🏻 
Not entirely agree on the default notice here, I would say definitely something to think about for future.